addons/hr_timesheet/models/project_task.py

```diff
@@ -182,6 +184,28 @@ def _get_timesheet(self):
         # Is override in sale_timesheet
         return self.timesheet_ids
 
+    def _get_timesheets_per_task(self, subtasks_included=False):
```

Comment:
`subtasks_included` is never used, is it?

Reply:
Ah zut, indeed. 🙈